editing/edit-features-with-feature-linked-annotation/src/main/java/com.esri.samples.edit_features_with_feature_linked_annotation/EditAttributesDialog.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.edit_features_with_feature_linked_annotation;
+
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Dialog;
+import javafx.scene.control.TextField;
+
+import com.esri.arcgisruntime.data.Feature;
+
+/**
+ * Custom dialog for editing feature attributes.
+ */
+public class EditAttributesDialog extends Dialog<Feature>{
+
+  @FXML private TextField addressTextField;
+  @FXML private TextField streetNameTextField;
+  @FXML private ButtonType continueButton;
+
+  EditAttributesDialog(Feature selectedFeature){
+    FXMLLoader loader = new FXMLLoader(getClass().getResource("/edit_features_with_feature_linked_annotation/edit_attributes_dialog.fxml"));
+    loader.setRoot(this);
+    loader.setController(this);
+
+    setTitle("Edit Feature Attributes");
+
+    try {
+      loader.load();
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+
+    // populate text fields with current attribute values
+    addressTextField.setText(selectedFeature.getAttributes().get("AD_ADDRESS").toString());
+    streetNameTextField.setText(selectedFeature.getAttributes().get("ST_STR_NAM").toString());
+
+    // convert the result to an address and street name when the ok button is clicked.
```

Comment:
in in line javadoc comments (i.e. those not describing a method/class), full stops/periods aren't required.
```suggestion
    // convert the result to an address and street name when the ok button is clicked
```